server/routes/induction/update/previousWorkExperienceTypesUpdateController.ts

```diff
@@ -183,7 +190,7 @@ const addedPreviousWorkExperienceTypes = (
 
 /**
  * Given the current Induction and the [PreviousWorkExperienceTypesForm], returns `true` if both the Induction and the
- * [PreviousWorkExperienceTypesForm] inclyde a Previous Work Experience type of OTHER, and the it's value is different.
+ * [PreviousWorkExperienceTypesForm] inclyde a Previous Work Experience type of OTHER, and if it's value is different.
```

Comment:
There are a couple more typos - one of them involves an apostrophe! 😉

Reply:
Done